output/mapping/F02_2014.csv

```diff
@@ -130,9 +130,9 @@ xpath,label-key,index,comment,guidance
 /PROCEDURE/TIME_RECEIPT_TENDERS,time,,,Map to the time component of `tender.tenderPeriod.endDate`
 /PROCEDURE/DATE_DISPATCH_INVITATIONS,date,,,Map to the date component of `tender.secondStage.invitationDate`.
 /PROCEDURE/LANGUAGES,languages_allowed,IV.2.4,,""
-/PROCEDURE/LANGUAGES/LANGUAGE,,,https://github.com/open-contracting-archive/trade/tree/master/draft_extensions/tender_SubmissionLanguages,Map to `tender.submissionLanguages` (PROPOSAL #23)
-/PROCEDURE/DATE_TENDER_VALID,date_tender_valid,,,Map to `tender.bidValidityPeriod.endDate` (PROPOSAL #33)
-/PROCEDURE/DURATION_TENDER_VALID,duration_months,,,"Multiply by 30, and map to `tender.bidValidityPeriod.durationInDays` (PROPOSAL #33)"
+/PROCEDURE/LANGUAGES/LANGUAGE,,,https://github.com/open-contracting-archive/trade/tree/master/draft_extensions/tender_SubmissionLanguages,Lowercase and add to the `tender.submissionTerms.language` array
```

Comment:
Change the extension to plural `languages`.

Reply:
Regarding the curly quotes, that must be LibreOffice that applied some regional setting, because I have to way of doing curly quote with my keyboard. I'll make sure it doesn't happen anymore.